src/device-registry/app.js

```diff
@@ -16,8 +16,8 @@ const { runKafkaConsumer, runKafkaProducer } = require("./config/kafkajs");
 
 mongodb;
 
-runKafkaProducer();
-runKafkaConsumer();
+// runKafkaProducer();
```

Comment:
just disabling the Kafka initialisation in the microservice since it is throwing continuous errors. @NoahNsimbe